README.md

```diff
@@ -50,12 +50,13 @@ Tested with the following:
 
 #### Required
 
+* `letsencrypt_certbot_default_args` - Required command line args to make Certbot work.
```

Comment:
As mentioned in the larger comment, this shouldn't be a "required" since it's not really - the defaults *are* there.